### PR TITLE
Create stage4 view and view-model

### PR DESCRIPTION
**Pull request by Peartes**
[X] Created the view for the stage4 component and make this view match the spec
[x] Implemented validation on the stage

### CODE REVIEW

File: src/newSeed/baseStage.scss [hunk 1]
```diff
@@ -1,6 +1,11 @@
 @import "styles/colors";
```
Comment (dkent600) [inline src/newSeed/baseStage.scss:1]: replace with

`@import "styles/styles.scss"`

File: src/newSeed/stage4.html [hunk 2]
```diff
@@ -1,5 +1,167 @@
...
+              placeholder="DD-MM-YYY" class="vestingDateInput" disabled>
+            <div><i class="far fa-calendar-alt" ref="startDate"></i></div>
+          </div>
+        </div>
+        <div>
+          <label for="startTime">Start Time</label>
+          <div class="vestingDatesContainer">
+            <div class="vestingDates">
+              <input id="startTime" type="time"
+              value.bind="seedConfig.seedDetails.startTime" placeholder="13:37 (UTC)"></numeric-input>
```
Comment (dkent600) [inline src/newSeed/stage4.html:71]: make it:  "00:00 (UTC)"

**Review by dkent600 — CHANGES_REQUESTED** [review]

File: src/newSeed/seedConfig.ts [hunk 3]
```diff
@@ -43,17 +43,37 @@ export interface ITokenDetails {
...
 
+export interface ISeedDetails {
+  seedTokens: BigNumber,
+  pricePerToken: BigNumber,
+  baseCurrency: string,
+  seedTarget: BigNumber,
+  seedMax: BigNumber,
+  vestingDays: number,
+  vestingCliff: number,
+  startDate: string,
```
[the remark this answers is not in the document]
Comment (dkent600) [reply src/newSeed/seedConfig.ts:54]: (by "I don't think we should have separate fields" I am referring to this stageCOnfig object, not the UI)

File: src/newSeed/seedConfig.ts [hunk 4]
```diff
@@ -43,17 +43,37 @@ export interface ITokenDetails {
   }>
 }
 
+export interface ISeedDetails {
+  seedTokens: BigNumber,
```
[the remark this answers is not in the document]
Comment (dkent600) [reply src/newSeed/seedConfig.ts:47]: Created a card here: https://www.notion.so/curvelabs/All-BigNumber-fields-in-ISeedConfig-should-be-strings-804afa1787e74283bd431c8925631ffd

[the remark this answers is not in the document]
Comment (Peartes) [reply src/newSeed/seedConfig.ts:47]: Okay

File: src/newSeed/stage4.html (hunk 2 above)
Comment (dkent600) [reply src/newSeed/stage4.html:71]: Sorry, should be:  00:00 (GMT)

**Review by dkent600 — APPROVED** [review]

**Pull request merged**